src/sentry/static/sentry/app/views/onboarding/projectSetup/firstEventIndicator.jsx

```diff
@@ -0,0 +1,146 @@
+import {keyframes} from 'emotion';
+import PropTypes from 'prop-types';
+import React from 'react';
+import posed, {PoseGroup} from 'react-pose';
+import styled from 'react-emotion';
+
+import {t} from 'app/locale';
+import Button from 'app/components/button';
+import EventWaiter from 'app/views/onboarding/projectSetup/eventWaiter';
+import InlineSvg from 'app/components/inlineSvg';
+import space from 'app/styles/space';
+import testablePose from 'app/utils/testablePose';
+
+const FirstEventIndicator = props => (
+  <EventWaiter {...props}>
+    {({firstIssue}) => <Indicator firstIssue={firstIssue} {...props} />}
+  </EventWaiter>
+);
+
+FirstEventIndicator.propTypes = {
+  orgId: PropTypes.string,
```

Comment:
This isn't used

Reply:
I think it ends up used in the EventWaiter, but in that case I should just set FirstEventIndicator.propTypes = EventWaiter.propTypes

Reply:
449c0c7